mapper/sort_map_slice.go

```diff
@@ -0,0 +1,55 @@
+package mapper
+
+import (
+	"sort"
+
+	"github.com/rancher/norman/types"
+	"github.com/rancher/norman/types/convert"
+)
+
+type SortMapSlice struct {
+	Field        string
+	SorterKey    string
+	ReverseOrder bool
```

Comment:
You only ever set reverOrder to true. Why over-complicate by adding this boolean?